src/mpc-hc/CMPCThemeUtil.h

```diff
@@ -54,7 +55,7 @@ class CMPCThemeUtil
     static CSize GetTextSizeDiff(CString str, HDC hDC, CWnd* wnd, int type, CFont* curFont);
 
 
-    static void GetMetrics(NONCLIENTMETRICS* ncMetrics);
+    static void GetMetrics(bool reset = true);

```

Comment:
default value of reset should be `false`?

Reply:
fixed.